### PR TITLE
[#960] fix(dashboard): simplify dependency and correct the startup script

### PR DESCRIPTION
**Pull request by zuston**
### What changes were proposed in this pull request?

simplify dependency and correct the startup script

### Why are the changes needed?

For #960

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Needn't

### CODE REVIEW

**Comment by zuston** [comment]
PTAL @yl09099 @xianjingfeng

File: bin/start-dashboard.sh [hunk 1]
```diff
@@ -26,16 +26,14 @@ load_rss_env
 
 cd "$RSS_HOME"
 
-COORDINATOR_CONF_FILE="${RSS_CONF_DIR}/coordinator.conf"
+DASHBOARD_CONF_FILE="${RSS_CONF_DIR}/dashboard.conf"
```
Comment (yl09099) [inline bin/start-dashboard.sh:29]: This will add a dashboard.conf configuration file, and the documentation needs to be modified.

Comment (xianjingfeng) [inline bin/start-dashboard.sh:29]: At present, there is no `dashboard.conf` in the tarball. Should we add a default file for this?

**Comment by zuston** [comment]
> I found other problems:
> 
> 1. We must set `HADOOP_HOME`
> 2. `Invalid process name: uniffle-dashboard` thrown by `get_pid_file_name` in utils.sh

Got it. Let me close this. I think @yl09099 will has more context to finish this.

**Pull request closed without merging**